doc/advanced-configuration/outgoing-connections.md

```diff
@@ -0,0 +1,208 @@
+# Outgoing connections
+
+MongooseIM can be configured to talk to external service like databases or HTTP servers in order to get or set required data.
+The interface for outgoing connections management was unified and is now available via the `outgoing_pools` config option for the following type of connections:
+
+* `cassandra` - pool of connections to cassandra cluster
+* `riak` - pool of connections to riak cluster
+* `redis` - pool of connections to redis server
+* `http` - pool of connections to various HTTP(S) server MongooseIM can talk to, in example HTTP auth backend or HTTP notifications
+* `generic` - pool of generic workers not assosiated directly with a paritcualr connection (SNS, PushNotifications)
+
+All the above pools are managed by [inaka/worker_pool](https://github.com/inaka/worker_pool) library.
+
+Every entry in the `outgoing_pools` is a 5 element tuple like below:
+
+```erlang
+{Type, Host, Tag, PoolOptions, ConnectionOptions}
+```
+
+Where:
+
+* `Type` is one of the types listed above
+* `Host` can be set to:
+    * `global` - meaning the pool will started once despited the XMPP hosts served by MongooseIM
```

Comment:
`global` - meaning the pool will be started once no matter how many XMPP hosts are served by MongooseIM